build.sh

```diff
@@ -278,14 +278,16 @@ case "$COMMAND" in
         ;;
 
     "ios-dynamic")
-        build_combined "iOS Dynamic" Realm "-dynamic" "LD_DYLIB_INSTALL_NAME='@rpath/RealmSwift.framework/Frameworks/Realm.framework/Realm'"
+        build_combined "iOS Dynamic" Realm "-dynamic"
         exit 0
         ;;
 
     "ios-swift")
+        sh build.sh ios-dynamic
```

Comment:
Why wasn't this step necessary prior to this change? Doesn't the shell script phase in RealmSwift's framework target already run `build.sh ios-dynamic` if the framework doesn't exist?